README.md

```diff
@@ -136,6 +136,33 @@ On Windows: `%APPDATA%/Claude/claude_desktop_config.json`
 
 </details>
 
+<details>
+  <summary>Docker-based Configuration</summary>
+
+Run `docker build -t mcp-server-docker:latest -f Dockerfile .`
+
+And then use this config:
+```
+"mcpServers": {
+  "mcp-server-docker": {
+    "command": "docker",
+    "args": [
+      "run",
+      "-i",
+      "--rm",
+      "-e",
+      "DOCKER_HOST",
+      "mcp-server-docker:latest"
+    ],
+    "env": {
+      "DOCKER_HOST": "tcp://<YOUR LOCAL IP>:2375"
+    }
+  }
+}
```

Comment:
I'd prefer the example uses the default Docker mount `/var/run/docker.sock` instead of setting the DOCKER_HOST env var:

```suggestion
"mcpServers": {
  "mcp-server-docker": {
    "command": "docker",
    "args": [
      "run",
      "-i",
      "--rm",
      "-v",
      "/var/run/docker.sock:/var/run/docker.sock",
      "mcp-server-docker:latest"
    ]
  }
}
```